src/clj/harja/kyselyt/sanktiot.sql

```diff
@@ -128,7 +127,119 @@ WHERE
         AND (lp.yllapitokohde IS NULL
         OR
              lp.yllapitokohde IS NOT NULL AND
-             (SELECT poistettu FROM yllapitokohde WHERE id = lp.yllapitokohde) IS NOT TRUE);
+             (SELECT poistettu FROM yllapitokohde WHERE id = lp.yllapitokohde) IS NOT TRUE)
```

Comment:
Saisiko tämän yksinkertaistettua riville 114 ->
LEFT JOIN yllapitokohde ypk ON lp.yllapitokohde = ypk.id AND ypk.poistettu IS NOT TRUE

Ja vielä parempi, jos poistettu kentälle antaisi default arvon ja pakottaisi arvot olemaan joko true tai false,
Ja sitten voisi tehdä vertailu ypk.poistettu = false. (tämä on suorituskyvyltään parempi)

Reply:
Poistettu-check ei toimi joinissa, koska silloin rivi näkyy, vaikka sen ypk olisi poistettu. Ilmeisesti siinä tapauksessa ei haluta näyttää sanktiota ollenkaan.

is not true -> is false voidaan tehdä, sillä on jo default false.